autoware_launch/config/localization/ekf_localizer.param.yaml

```diff
@@ -22,6 +22,11 @@
     proc_stddev_vx_c: 10.0
     proc_stddev_wz_c: 5.0
 
+    #Simple1DFilter parameters
+    z_filter_proc_dev: 1.0
+    roll_filter_proc_dev: 0.01
+    pitch_filter_proc_dev: 0.01
+    
```

Comment:
There are unnecessary whitespaces. It is causing pre-commit violations.  Please remove them.

```suggestion

```

About pre-commit: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/ci-checks/#pre-commit